spec/requests/api/v1/targets/index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe 'GET api/v1/targets', type: :request do
+  let(:user) { create(:user) }
+  let(:headers) { auth_headers }
+  let(:parsed_response) { JSON.parse(response.body) }
+
+  subject { get api_v1_targets_path, as: :json, headers: headers }
```

Comment:
We can use `subject!` with a `!` so it's called automatically, and we can avoid adding the `before` blocks for calling it